### DISCUSSION TITLE
[3 Novembre 2022] Modifica CPU

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/architettura-degli-elaboratori#67] [Modifiche architettura MIPS (M-Z)]
![Esame](https://github.com/Elia-Belli/ae2223/assets/107944476/4511d4ee-840c-41b0-a295-2daf017692db)
![Esame](https://github.com/Elia-Belli/ae2223/assets/107944476/b01d6242-ada8-4fa3-b216-3cb764dc7e0c)
![CPU](https://github.com/Elia-Belli/ae2223/assets/107944476/d8a4dc04-dce1-44f1-854b-0f7caf2914b9)
---
### Soluzione del Prof
Le soluzioni fornite dal professore potrebbero essere incomplete o poco chiare, quindi sono benvenute soluzioni che le completano, oppure soluzioni alternative e/o con spiegazioni più approfondite: 
- [Disegno.pdf](https://github.com/sapienzastudentsnetwork/archelab2223/files/11665964/AE.2021-22.-.Appello.S1.-.2022-11-03.-.Esercizio.CPU.-.diagramma.pdf)
- [Soluzione.pdf](https://github.com/sapienzastudentsnetwork/archelab2223/files/11665965/AE.2021-22.-.Appello.S1.-.2022-11-03.-.Esercizio.CPU.-.Scheda.di.calcolo.pdf)

**Comment by Frisbon** [comment]
La mia soluzione (che spero si capisca + di quella del prof) + i dettagli e ragionamenti x ogni punto.

[Bro.. Ti sei cagato addosso...](https://github.com/sapienzastudentsnetwork/archelab2223/files/11855931/Bro.Ti.sei.cagato.addosso.pdf)

**Comment by daromacarlo** [comment]
![Catturdddddddddddddda](https://github.com/sapienzastudentsnetwork/archelab2223/assets/135441131/80930afb-f294-445d-9f70-326380e5d172)
